packages/flutter/test/material/text_theme_test.dart

```diff
@@ -152,4 +152,58 @@ void main() {
     expect(sizeTheme.overline.fontSize, baseTheme.overline.fontSize * 2.0 + 5.0);
   });
 
+  test('TextTheme lerp with second parameter null', () {
+    final TextTheme theme = Typography(platform: TargetPlatform.android).black;
```

Comment:
Suggestion: `platform` defaults to `TargetPlatform.android` and we don't really care anyways for the context of these tests.

I would just do `Typography().black` or `Typography.blackMountainView`

Reply:
Done